frontend/app/src/pages/user/create/index.vue

```diff
@@ -1,21 +1,142 @@
 <script setup lang="ts">
-import Fragment from '@/components/helper/Fragment';
+const { appVersion } = storeToRefs(useMainStore());
 
 const { upgradeVisible } = storeToRefs(useSessionAuthStore());
 const { navigateToUserLogin } = useAppNavigation();
 const { createNewAccount, error, loading } = useAccountManagement();
+
+const { t } = useI18n();
+const css = useCssModule();
+
+const step: Ref<number> = ref(1);
+const steps = [
+  {
+    title: t('create_account.steps.step_1.title'),
+    description: t('create_account.steps.step_1.description')
+  },
+  {
+    title: t('create_account.steps.step_2.title'),
+    description: t('create_account.steps.step_2.description')
+  },
+  {
+    title: t('create_account.steps.step_3.title'),
+    description: t('create_account.steps.step_3.description')
+  },
+  {
+    title: t('create_account.steps.step_4.title')
+  }
+];
 </script>
 
 <template>
-  <Fragment>
-    <UpgradeProgressDisplay v-if="upgradeVisible" />
-    <CreateAccountForm
-      v-else
-      :loading="loading"
-      :error="error"
-      @error:clear="error = ''"
-      @cancel="navigateToUserLogin()"
-      @confirm="createNewAccount($event)"
-    />
-  </Fragment>
+  <div :class="css.overlay">
+    <div :class="css.overlay__scroll">
+      <section :class="css.section">
+        <div :class="css.container">
+          <div :class="css.wrapper">
+            <div class="pb-4" data-cy="account-management">
+              <UserHost>
```

Comment:
this part seems similar with `login/index.vue` do youthink we can extract some common css + login?